ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -584,4 +592,44 @@ public void setHitSlopRect(@Nullable Rect rect) {
     mHitSlopRect = rect;
   }
 
+  public void setOpacityIfPossible(float opacity) {
+    mBackfaceOpacity = opacity;
+    setBackfaceVisibilityDependantOpacity();
+  }
+
+  public void setBackfaceVisibility(String backfaceVisibility) {
+    mBackfaceVisibility = backfaceVisibility;
+    setBackfaceVisibilityDependantOpacity();
+  }
+
+  public void decomposeMatrix(ReadableArray matrix) {
```

Comment:
Could we just use View#getRotationX and View#getRotationY instead of keeping our own transform info here?

Reply:
Good point!